gui/default/syncthing/folder/restoreVersionsModalView.html

```diff
@@ -1,4 +1,4 @@
-<modal id="restoreVersions" status="default" icon="fas fa-undo" heading="{{'Restore Versions' | translate}}" large="yes" closeable="yes">
```

Comment:
gui changes should still not be in this PR

Reply:
It's on another branch. "dialog-titre"